apps/prairielearn/assets/scripts/instructorAssessmentManualGradingInstanceQuestion.js

```diff
@@ -0,0 +1,515 @@
+import ClipboardJS from 'clipboard';
+
+$(() => {
+  resetInstructorGradingPanel();
+
+  document.addEventListener('keypress', (event) => {
+    // Ignore holding down the key events
+    if (event.repeat) return;
+    // Ignore events that target an input element
+    if (
+      !['TEXTAREA', 'SELECT'].includes(event.target.tagName) &&
+      (event.target.tagName !== 'INPUT' ||
+        ['radio', 'button', 'submit', 'checkbox'].includes(event.target.type)) &&
+      !event.target.isContentEditable
+    ) {
+      document
+        .querySelectorAll(
+          `.js-selectable-rubric-item[data-key-binding="${event.key}"]:not(:disabled)`
+        )
+        .forEach((item) => item.dispatchEvent(new MouseEvent('click')));
+    }
+  });
+  const modal = document.querySelector('#conflictGradingJobModal');
+  if (modal) {
+    $(modal)
+      .modal({})
+      .on('shown.bs.modal', function () {
```

Comment:
I think this can be replaced by:
```
document.querySelector('#conflictGradingJobModal')?.addEventListener('show.bs.modal', (e) => {
    const modal = e.target as HTMLElement;
```

Reply:
The modal creation requires Bootstrap v4 anyway, so adding the event listener in vanilla JS would not be a big improvement since we already have the jQuery object anyway.

Reply:
Sure, I'm fine with this.